tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -41,4 +41,8 @@ apt-get update
 apt-get install -y rats-tls
 EOF
 	fi
+
+        # Integrate pause image inside rootfs
+        skopeo copy docker://k8s.gcr.io/pause:3.6 oci:pause:3.6
```

Comment:
The `pause` image will be an important asset in the guest image; I suggest to put its url and version in the `versions.yaml` so that 1) can be easily bumped, 2) ensure same version/asset used on other distro guests

Reply:
Thanks @wainersm , agree, I'll use `versions.yaml` to track `pause` image